README.md

```diff
@@ -21,7 +21,6 @@ Tightening of any checks means breaking change and should be published with majo
 - **Patch:** Fixing of minor oversights
 
 ## How to publish new version
-- Check [NPM package publishing guide](https://github.com/outfunnel/outfunnel/wiki/Publishing-NPM-libraries)
+- Check [NPM package publishing guide](https://www.notion.so/outfunnel/Publishing-NPM-libraries-ada823f30d60489abe091b86b39594a4)
```

Comment:
This is public repo. Does it make sense to put private link here?

Reply:
true, makes sense to remove. one sec